packages/config-array/rollup.std__path-config.js

```diff
@@ -0,0 +1,28 @@
+export default [
+	{
+		input: "../../node_modules/@jsr/std__path/posix/mod.js",
```

Comment:
Would it be possible to make this not depend on where the package is installed and also not depend on its internal structure?

Reply:
Done in dd0d81bc55cbe6c7b035af06cab5a7f587b85324.